app/src/components/fields/StartEndDateFields.tsx

```diff
@@ -43,6 +44,7 @@ const StartEndDateFields: React.FC<IStartEndDateFieldsProps> = (props) => {
         <TextField
           id="end_date"
           name="end_date"
+          data-testid="end-date"
```

Comment:
question:
why did this get added? I dont see it being used in any of the tests

Reply:
I'm going to use the test id's to find these fields and click on them ..  this is all in order to enter the else statement in the  projectfundingform.

Reply:
Ah gotcha, should it be added in the subsequent PR then?

Reply:
I intended to do it in this PR, but yes ... let's get this PR in, I'll add the clicks to the next PR